Algorithm.CSharp/BuyBTCWithLessThanOneDollarAlgorithm.cs

```diff
@@ -0,0 +1,134 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using QuantConnect.Orders;
+using QuantConnect.Interfaces;
+using QuantConnect.Data;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    public class BuyBTCWithLessThanOneDollarAlgorithm : QCAlgorithm, IRegressionAlgorithmDefinition
+    {
+        public override void Initialize()
+        {
+            SetStartDate(2013, 10, 1);
+            SetEndDate(2013, 10, 1);
+            SetCash(1);
+            SetBrokerageModel(Brokerages.BrokerageName.Bitfinex, AccountType.Cash);
+            AddCrypto("BTCUSD", Resolution.Minute, Market.Bitfinex);
```

Comment:
> Market.Bitfinex

Specifying the market shouldn't be required, let's remove it 👌. Default resolution is already Minute too